README.adoc

```diff
@@ -11,12 +11,13 @@ projects: []
 :toc:
 :icons: font
 :source-highlighter: prettify
-:project_id: draft-gs-template
+:project_id: gs-spring-cloud-gcp
 This guide walks you through the process of creating a Spring application.
```

Comment:
Also, I went for spring-cloud-gcp to describe the project thoughout the guide, although it only applies to Spring Integration. I suggest to rename when we publish other guides.